test/fixtures/get-orderbook-response.json

```diff
@@ -324,9 +324,9 @@
           "value": "0.5",
           "counterparty": "rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B"
         },
-        "totalPrice": {
+        "price": {
```

Comment:
Wait I was confusing myself anyway I think? output_rate is not really right
huh?

exchangeRate might work. It would always be what is named quality ?

Basically, I think ideally the quality, uninverted would be available for
whatever math. If you can find a good name for it ?